cmd/kube-proxy/app/options/options.go

```diff
@@ -63,25 +63,25 @@ func NewProxyConfig() *ProxyServerConfig {
 func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.BindAddress}, "bind-address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
-	fs.IntVar(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
+	fs.Int32Var(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
```

Comment:
@smarterclayton - all the changes int<->int32 are pretty independent and we can do them very safely.
Can you please move them to a separate PR - so that we can merge it earlier, since this doesn't seem to be controversial.